ToDoList/src/features/toDoSlice.js

```diff
@@ -0,0 +1,83 @@
+/* eslint-disable no-param-reassign */
+/* eslint-disable no-console */
+import { createSelector, createSlice } from '@reduxjs/toolkit';
+import { StatusFilters } from './filterSlice';
+import 'regenerator-runtime/runtime';
+
+const initialState = {
+  toDoList: [],
+  status: 'idle',

```

Comment:
можно использовать флаг `isLoading`
```suggestion
  isLoading: false,

```